<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DataSetSelectorWidgetForm</class>
 <widget class="QWidget" name="DataSetSelectorWidgetForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>474</width>
    <height>274</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1">
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="m_datasetGroupBox">
     <property name="title">
      <string>DataSource Title</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="1,1">
     <item>
      <widget class="QGroupBox" name="m_mapPreviewGroupBox">
       <property name="title">
        <string>Map Preview</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="m_dataPreviewGroupBox">
       <property name="title">
        <string>Data Preview</string>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
